airflow/models/__init__.py

```diff
@@ -2805,6 +2806,18 @@ class DagModel(Base):
     def __repr__(self):
         return "<DAG: {self.dag_id}>".format(self=self)
 
+    @property
+    def _schedule_interval(self):
+        """'Normalized' schedule interval, which is either a timedelta or a cron
+           interval (i.e. does not contain special intervals such as @once or @daily).
+        """
+        interval = self.schedule_interval
+        if isinstance(interval, Hashable) and interval in cron_presets:
```

Comment:
In this case, the `interval` is just a `str`. Why Hashable? I would implement this as `isinstance(interval, six.string_types)`: https://pythonhosted.org/six/#six.string_types

Reply:
Same as above, this method was moved from the `DAG` class. I don't know if there was a specific reason for using hashable?